lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -950,11 +1040,19 @@ sub get_dataset_allele_response {
 
     my @unique_genes = uniq @genes;
     $result_details->{MolecularAttributes}->{geneIds} = \@unique_genes if (scalar @unique_genes > 0);
+    $result_details->{MolecularAttributes}->{geneOntology} = $gene_ontology if (scalar @{$gene_ontology} > 0);
```

Comment:
`geneOntology` is not part of the spec, is it okay to include it?

Reply:
This is another update I'm going to request https://github.com/ga4gh-beacon/beacon-v2